src/config/constants.js

```diff
@@ -0,0 +1,14 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
```

Comment:
Could you destructure these variables from `process.env`

Reply:
Thanks @jsamchineme for the feedback. I will implement them now.

Reply:
> Having reviewed what you have done i think this is well written code you have done well. Just a suggestion tho, since the variables in the config file are suppose to be constants by convention constants are usually all capital letter variable name.

Thanks @orley12 , I have done that.